src/django/api/views/v1/opensearch_query_builder/moderation_events_query_builder.py

```diff
@@ -55,7 +55,7 @@ def add_sort(self, field, order_by=None):
                 {
                     'cleaned_data.' +
                     field +
-                    '.alpha_2': {'order': order_by}
+                    '.name': {'order': order_by}
```

Comment:
Hi @roman-stolar 
It looks like this `if block` is related to sorting by country. Why should we change it to sorting by `name` instead of `alpha_2` when the `field` is equal to `V1_PARAMETERS_LIST.COUNTRY`?